docs/refguide/computed-with-args.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Computeds with arguments
+sidebar_label: Computeds with arguments 🚀
+hide_title: true
+---
+
+<script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?serve=CEBD4KQ7&placement=mobxjsorg" id="_carbonads_js"></script>
+
+## Computeds with arguments
+
+The `computed` annotation can only be used on getters, which don't take arguments.
+What about computations that do take arguments?
+Take the below example of a React component that renders a specific Item,
+and the application supports multi-selection.
+How can we implement a derivation like `store.isSelected(item.id)`?
+
+Base example (based on React):
+
+```javascript
+import * as React from 'react'
+import {observer} from 'mobx-react-lite'
+
+const Item = observer(({ item, store }) => (
+    <div className={store.isSelected(item.id) ? "selected" : ""}>
+        {item.title}
+    </div>
+)
+```
+
+There are four ways in which we can approach this. You can try the solutions below here: [sandbox](https://codesandbox.io/s/multi-selection-odup1?file=/src/index.tsx).
+
+### 1. Derivations don't _need_ to be `computed`
+
+A function doesn't need to be marked `computed` to enable to MobX to track it.
+The above example would already work completely fine out of the box.
+It is important to realise that computed values are only _caching points_.
+If they are pure (and they shouldn't), having a getter or function without `computed` doesn't change behavior; it is just slightly less inefficient.
+The above example works fine despite `isSelected` not being a `computed`;
+the `observer` component will detect and subscribe to any observables that were read by `isSelected`, because the function executes as part of the rendering that is tracked.
```

Comment:
```suggestion
the `observer` component will detect and subscribe to any observables that were read by `isSelected` because the function executes as part of the rendering that is tracked.
```